app/assets/stylesheets/application.scss

```diff
@@ -1 +1,30 @@
 @import 'bootstrap';
+
+.myButton {
+	background-color:#44c767;
+	-moz-border-radius:28px;
+	-webkit-border-radius:28px;
+	border-radius:28px;
+	border:1px solid #18ab29;
+	display:inline-block;
+	cursor:pointer;
+	color:#ffffff;
+	font-family:Arial;
+	font-size:17px;
+	padding:16px 31px;
+	text-decoration:none;
+  text-shadow:0px 1px 0px #2f6627;
+  justify-content: flex-end;
+  align-items: flex-end;
+  float: right;
+  margin-right: 20px;
+}
+.myButton:hover {
+	background-color:#5cbf2a;
+}
+.myButton:active {
+	position:r;
+	top:1px;
+}
+
+        
```

Comment:
I don't know how it happened, but there are two unnecessary empty line at the end of the file, but GitHub whines about no newline character at the end of file 🤯